go/teams/transactions.go

```diff
@@ -113,8 +113,6 @@ func (tx *AddMemberTx) createInvite(ctx context.Context, uv keybase1.UserVersion
 		payload.Admins = appendToInviteList(invite, payload.Admins)
 	case keybase1.TeamRole_OWNER:
 		payload.Owners = appendToInviteList(invite, payload.Owners)
```

Comment:
Still too scary. Can it just return error in a `default` case here?

Reply:
Right, this is not done yet... I kind of got distracted by other tickets and didn't fix everything in here because it's not obvious. The issue is that if this fails, then line `addMemberByUPKV2` will return an error on line 237 but that will be after the `AddMemberTx` object was already modified. But the contract is that if any of `AddMemberTx` API functions returns an error then call had no effect on tx object.

Right play would be to change the whole thing to operate on copies of objects, then situations like these would be easier to handle. I'll get back to this PR.

Reply:
Oh! I see. That is annoying. You could panic.. that might not be better